scripts/StartDB1000N.bat

```diff
@@ -3,21 +3,21 @@ rem Parsing command line argument to get target download folder
 echo $SaveFolder = $args[0]                                                       >  %temp%\GetDB1000N.ps1
 echo if ($args.count -lt 1)                                                       >> %temp%\GetDB1000N.ps1
 echo {                                                                            >> %temp%\GetDB1000N.ps1
-echo 	Write-Host "Missing save folder parameter"                                >> %temp%\GetDB1000N.ps1
+echo 	Write-Host "Missing save folder parameter"                                  >> %temp%\GetDB1000N.ps1
```

Comment:
Can you check whitespace changes here? this one seems off

Reply:
Hi,

You are right, this is only a whitespace I must have add by error.

It can be deleted.

Reply:
you must do it in your branch in order for it to get into the PR

Reply:
Hi,

Hope I done it correctly as I'am not used in usng GitHub.